planner/core/stats_test.go

```diff
@@ -139,7 +139,7 @@ func (s *testStatsSuite) TestGroupNDVs(c *C) {
 	}
 }
 
-func (s *testStatsSuite) TestCardinalityGroupCols(c *C) {
+func (s *testStatsSuite) TestNDVGroupCols(c *C) {
```

Comment:
This change breaks the test. The "Name" in the json file also needs to be changed.